cndocs/debugging-native-code.md

```
@@ -10,29 +10,70 @@ import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import con
 npx react-native log-ios
 ```
 
-你也可以通过 iOS 模拟器中的 "Debug > Open System Log..." 菜单，或者在运行 Android 应用的设备或模拟器中运行 `adb logcat "*:S" ReactNative:V ReactNativeJS:V` 来访问这些日志。
+你也可以通过 iOS 模拟器中的 Debug > Open System Log… 菜单，或者在运行 Android 应用的设备或模拟器中执行 `adb logcat "*:S" ReactNative:V ReactNativeJS:V` 来查看这些日志。
+
+<details>
+<summary>**💡 自定义原生日志**</summary>
+
+如果你正在编写原生模块，并且想为你的模块添加自定义日志用于调试，可以使用以下方法：
+
+#### Android（Java/Kotlin）
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)